tests/cli/test_export_cli.py

```diff
@@ -152,6 +152,57 @@ def test_dynamic_batching(self):
                 check=True,
             )
 
+    @requires_neuronx
+    def test_opt_level(self):
+        model_id = "hf-internal-testing/tiny-random-BertModel"
+        optlevels = ["-O1", "-O2", "-O3"]
+        for optlevel in optlevels:
+            with tempfile.TemporaryDirectory() as tempdir:
+                subprocess.run(
+                    [
+                        "optimum-cli",
+                        "export",
+                        "neuron",
+                        "--model",
+                        model_id,
+                        "--sequence_length",
+                        "16",
+                        "--batch_size",
+                        "1",
+                        "--task",
+                        "text-classification",
+                        optlevel,
+                        tempdir,
+                    ],
+                    shell=False,
+                    check=True,
+                )
+
+    @requires_neuronx
+    def test_store_intemediary(self):
+        model_id = "hf-internal-testing/tiny-random-BertModel"
+        with tempfile.TemporaryDirectory() as tempdir:
+            subprocess.run(
+                [
+                    "optimum-cli",
+                    "export",
+                    "neuron",
+                    "--model",
+                    model_id,
+                    "--sequence_length",
+                    "16",
+                    "--batch_size",
+                    "1",
+                    "--task",
+                    "text-classification",
+                    "--compiler_workdir",
+                    f"{tempdir}/neff",
+                    tempdir,
+                ],
+                shell=False,
+                check=True,
+            )
```

Comment:
Maybe here you could check that there is a NEFF file in the tempdir/neff directory.

Reply:
sounds good, I will add a check for that!